modules/metastore/metastore.routing.yml

```diff
@@ -165,7 +165,7 @@ metastore.data_dictionary:
     _controller: '\Drupal\system\Controller\SystemController::systemAdminMenuBlockPage'
     _title: 'Data-Dictionary'
   requirements:
-    _permission: 'administer site configuration'
+    _permission: 'create data content'
 
 metastore.data_dictionary.settings:
   path: '/admin/dkan/data-dictionary/settings'
```

Comment:
This route is in both metastore.routing.yml and datastore.routing.yml

Reply:
So guessing the more global DKAN administration permission will be a different effort.... and this PR is just to get the DD links available?

Reply:
Yes, that is exactly right. This was just to get the DD links available.